app/routes/cards_routes.py

```diff
@@ -0,0 +1,71 @@
+from app import db
+from flask import Blueprint, abort, make_response, request, jsonify
+from app.models.card import Card
+from app.models.board import Board
+
+
+cards_bp = Blueprint("cards", __name__, url_prefix="/boards/<board_id>/cards")
+
+def validate_model(cls, model_id):
```

Comment:
This duplicates code from `board_routes_helper.py`, we could import the existing function from the helper file and use it here.